src/neo/LogLevel.cs

```diff
@@ -2,10 +2,10 @@ namespace Neo
 {
     public enum LogLevel : byte
     {
-        Fatal,
-        Error,
-        Warning,
-        Info,
-        Debug
+        Debug = 0,
+        Info = 1,
+        Warning = 2,
+        Error = 3,
+        Fatal = 4
```

Comment:
Why modify the order?

Reply:
Because it's easier if we want to log `Error` and `Fatal`, we can use > `Error`